render_pdf/src/make_latex.py

```diff
@@ -0,0 +1,92 @@
+from typing import Union
+import parser
+import os
+import jinja2
+
+# TODO probably best to use pydantic for a config template for the dict below. 
+data = {
+        'language' : "sanskrit",
+        'title' : "",
+        'author' : "",
+        'body' : [],
+        'analysis': []
+    }
+
+os.chdir("../tex_files")
+latex_jinja_env = jinja2.Environment(
+        block_start_string = '((*',
+        block_end_string = '*))',
+        variable_start_string = '(((',
+        variable_end_string = ')))',
+        comment_start_string = '((=',
+        comment_end_string = '=))',
+        loader = jinja2.FileSystemLoader(os.path.abspath('.'))
+    )
+template = latex_jinja_env.get_template('template.tex')
+
+## TODO add an argument for receiving language, default is sanskrit + devanagari
+
+def write_tex(tex: str):
+    os.chdir("../tex_files")
+    with open("./template.tex", 'w') as f:
+        f.write(tex)
+
+def generate_fields(link: str, lipi: Union["SKT","IAST"], author: str) -> tuple[list, str, str]:
```

Comment:
nit: DEVA instead of SKT ?

Reply:
Done, but this is a placeholder for once the transliterator is ready. I'll revisit and add in transliteration support then.